lib/msf/core/exploit/file_dropper.rb

```diff
@@ -201,6 +201,10 @@ def file_dropper_delete_file(session, file)
   # @param [String] dir The directory to delete
   # @return [Boolean] True if the delete command has been executed in the remote machine, otherwise false.
   def file_dropper_delete_dir(session, dir)
+    if file_dropper_check_cwd?(dir)
+      print_warning("Attempting to delete working directory #{dir}")
```

Comment:
I kind of want to add a "tip" after this warning to tell users they can use the `cd` command.